include/raft.h

```diff
@@ -783,7 +783,9 @@ struct raft
             raft_index round_index;         /* Target of the current round. */
             raft_time round_start;          /* Start of current round. */
             void *requests[2];              /* Outstanding client requests. */
-            uint64_t reserved[8];           /* Future use */
+            uint32_t voter_contacts;        /* Current number of voting nodes we are in contact with */
```

Comment:
Please initialize this explicitly to 0 in raft_init, and reset it to 1 when entering leader state. (Without the second part, raft_voter_contacts could return a stale value from a previous term.)